tests/Backend/Synapse/CreateTableTest.php

```diff
@@ -103,4 +103,62 @@ public function testCreateTableWithCrlfLineEndings()
         $importedData = $this->_client->getTableDataPreview($tableId);
         $this->assertCount(5, Client::parseCsv($importedData));
     }
+
+    public function testCreateTableDefinition()
```

Comment:
Tu sme sa bavili ze by to slo otestovat tak, ze si skontrolujes ze sa vystrelila eventa pre ten job